### PR TITLE
feat: add frontend-app-ora-grading to devstack as an extra service

### PR DESCRIPTION
**Pull request by leangseu-edx**
add frontend-app-ora-grading to devstack as an extra service

Ticket: [AU-321](https://openedx.atlassian.net/browse/AU-321)
Related PR: [frontend-app-ora-grading](https://github.com/edx/frontend-app-ora-grading/pull/14)

- [x] add docker configuration
- [x] update document in README
- [x] update git repo url

### Test

- [x] `make dev.up` should not spin up `frontend-app-ora-grading`
- [x] `make dev.up.frontend-app-ora-grading` should spin up the `esg-mfe`
- [x] Basic make command
  - [x] `make dev.attach.frontend-app-ora-grading` work
  - [x] `make dev.shell.frontend-app-ora-grading` work
  - [x] `make dev.clone.frontend-app-ora-grading` work

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -33,6 +33,7 @@ It also includes the following extra components:
 * The Library Authoring micro-frontend
 * edX Registrar service.
 * The course-authoring micro-frontend
+* The enhanced staff grader (ora-grading) micro-frontend
```
Comment (muselesscreator) [inline README.rst:36]: I like this phrasing

File: repo.sh [hunk 2]
```diff
@@ -96,7 +98,7 @@ _checkout ()
 
         # If a directory exists and it is nonempty, assume the repo has been cloned.
         if [ -d "$name" ] && [ -n "$(ls -A "$name" 2>/dev/null)" ]; then
-            echo "Checking out branch ${OPENEDX_GIT_BRANCH} of $name"
+            echo "Change directory to $name"
```
Comment (muselesscreator) [inline repo.sh]: why remove reference to checking out the branch, when that is still happening?

Comment (leangseu-edx) [reply repo.sh]: Sorry that was an attempt to support `main` as default. It was part of different [PR](https://github.com/edx/devstack/pull/845). I revert this change back.

File: Makefile [hunk 3]
```diff
@@ -149,13 +149,13 @@ selfcheck: ## Check that the Makefile is free of Make syntax errors.
 # Developer interface: Repository management.
 ########################################################################################
 
-dev.reset-repos: ## Attempt to reset the local repo checkouts to the master working state.
+dev.reset-repos: ## Attempt to reset the local repo checkouts to the default branch working state.
```
Comment (timmc-edx) [inline Makefile]: Looks like there are still some changes leftover here from the PR -- perhaps inadvertently committed on this branch.

Comment (timmc-edx) [reply Makefile]: Oh, I guess it was the same PR. Anyway, some leftovers in this file. Everything else looks great.

Comment (leangseu-edx) [reply Makefile]: Good catch 👍

**Review by timmc-edx — APPROVED** [review]

**Review by jnlapierre — APPROVED** [review]
LGTM and works in my local devstack.

**Review by muselesscreator — APPROVED** [review]
LGTM

**Pull request merged**